### PR TITLE
Bunker remap

### PR DESCRIPTION
**Pull request by DreamySkrell**
image

image

image

- functionally should be same or better
- can accommodate more people - seats for 3 BCs, Captain, and XO at the very least
- more consoles and chairs in general
- emergency food and drink
- more free/unoccupied space in general to walk around or store things at
- ship and long-range holopads
- keycard authentication device

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

**Comment by Arrow768** [comment]
I am a bit unsure about the presence of the helm/weapons console.
Afaik we do not have the capability to transfer the control capability from one helm/weapons console to another.
Without that capability I am a bit concerned that this will lead to the bridge crew and a antag sitting in the bunker trying to take control of the ship.

**Comment by DreamySkrell** [comment]
> I am a bit unsure about the presence of the helm/weapons console. Afaik we do not have the capability to transfer the control capability from one helm/weapons console to another. Without that capability I am a bit concerned that this will lead to the bridge crew and a antag sitting in the bunker trying to take control of the ship.

I've not added any new instances of these consoles - they are all there on the live server currently, in the current bunker layout.

And like, yeah, I've not heard of fighting for control really being a problem. When or if it does happen, it should be fixed in another PR.

Also uh, the CIC has just one entrance, and is closed off, so it's probably very easy to assault for security, or chuck an EMP in, or something. Or even welder bomb the APC that is just beside the door lol.

**Review by Arrow768 — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

File: html/changelogs/DreamySkrell-bunker-remap-43267345.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - tweak: "Expands and tweaks the CIC bunker layout and interior."
```
Comment (SleepyGemmy) [inline html/changelogs/DreamySkrell-bunker-remap-43267345.yml:41]:
```suggestion
  - maptweak: "Expands and tweaks the CIC bunker layout and interior."
```

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**